en_US/advanced/relup.md

```diff
@@ -1,312 +1,173 @@
 
 ## Release Upgrade
 
-{% emqxce %}
+## Release Hot Upgrade
 
-Since version 4.2.0, EMQX supports hot upgrade.
+EMQX support release hot upgrade since version 4.2.0.
 
-By using the hot upgrade feature, users can quickly and safely upgrade the EMQX in the production environment, and avoid the decrease in system availability caused by restarting the service.
+With the release hot upgrade feature, users can quickly and safely upgrade the EMQX of the production environment while keeping all the clients connected, avoid reducing the system availability caused by restarting the service.
```

Comment:
```suggestion
With the release hot upgrade feature, users can quickly and safely upgrade EMQX in production environment while keeping all the clients connected, ensure high availability because there is no need to restart the service.
```